.github/workflows/backend-cd.yml

```
@@ -0,0 +1,50 @@
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest # 작업이 실행될 환경
+    defaults:
+      run:
+        working-directory: backend
+    steps:
+      - name: 체크아웃
+        uses: actions/checkout@v3
+      - name: JDK 17 사용
+        uses: actions/setup-java@v3
```

actionlint (1.7.4)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)